<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DelegatesPage</class>
 <widget class="QWidget" name="DelegatesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>532</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <stylestrategy>PreferDefault</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QScrollBar{background-color: rgb(255, 255, 255);}</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>146</x>
     <y>19</y>
     <width>3</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(187, 187, 187);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>58</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(134, 134, 134);</string>
   </property>
   <property name="text">
    <string>Rank</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>58</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(134, 134, 134);</string>
   </property>
   <property name="text">
    <string>Accounts</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>386</x>
     <y>58</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(134, 134, 134);</string>
   </property>
   <property name="text">
    <string>Votes</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>568</x>
     <y>58</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(134, 134, 134);</string>
   </property>
   <property name="text">
    <string>Supporting delegates</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>159</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(134, 134, 134);</string>
   </property>
   <property name="text">
    <string>Balance</string>
   </property>
  </widget>
  <widget class="QToolButton" name="nextPageBtn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>492</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pageLineEdit">
   <property name="geometry">
    <rect>
     <x>645</x>
     <y>497</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #282828;border:1px solid #536874;</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="numberLabel">
   <property name="geometry">
    <rect>
     <x>578</x>
     <y>492</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>page</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pageLabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>492</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #282828;</string>
   </property>
   <property name="text">
    <string>--/--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="prePageBtn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>492</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>51</y>
     <width>792</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(187,187,187);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="allDelegatesBtn">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>14</y>
     <width>102</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>All</string>
   </property>
  </widget>
  <widget class="QToolButton" name="supportedDelegatesBtn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>14</y>
     <width>102</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Supporting</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>98</y>
     <width>700</width>
     <height>380</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);
border:0px solid rgb(255, 255, 255);
border-right-width:1px;</string>
   </property>
  </widget>
  <widget class="QToolButton" name="goToBtn">
   <property name="geometry">
    <rect>
     <x>695</x>
     <y>495</y>
     <width>50</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(49,189,198);
border-radius:4px;</string>
   </property>
   <property name="text">
    <string>Go</string>
   </property>
  </widget>
  <widget class="QLabel" name="moveLabel">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>48</y>
     <width>100</width>
     <height>3</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(49,189,198);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="helpBtn">
   <property name="geometry">
    <rect>
     <x>705</x>
     <y>66</y>
     <width>14</width>
     <height>14</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>14</width>
     <height>14</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="helpTipsLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>3</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255,255,255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Mark the delegates you supported and complete the vote once the transfer is completed.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_12</zorder>
  <zorder>label_3</zorder>
  <zorder>label_9</zorder>
  <zorder>label_11</zorder>
  <zorder>label_13</zorder>
  <zorder>label_14</zorder>
  <zorder>nextPageBtn</zorder>
  <zorder>pageLineEdit</zorder>
  <zorder>numberLabel</zorder>
  <zorder>pageLabel</zorder>
  <zorder>prePageBtn</zorder>
  <zorder>label_15</zorder>
  <zorder>allDelegatesBtn</zorder>
  <zorder>supportedDelegatesBtn</zorder>
  <zorder>listView</zorder>
  <zorder>goToBtn</zorder>
  <zorder>moveLabel</zorder>
  <zorder>helpBtn</zorder>
  <zorder>helpTipsLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
